projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-color.service.ts

```diff
@@ -50,7 +46,7 @@ export class DataLayerColorService {
     return undefined;
   }
 
-  /** build a CSS linear gradient that includes colors from all datasets in the layer */
+  //  build a CSS linear gradient that includes colors from all datasets in the layer
   getColorGradient(layer: DataLayer) {
```

Comment:
why did you change this from a doc comment to a line comment?

Reply:
Okay, I will change the line comment to a doc comment.